src/layout.jl

```diff
@@ -174,7 +175,7 @@ print_block(io::IO, x::ChainBlock) = printstyled(io, "chain"; bold=true, color=c
 print_block(io::IO, x::ReflectGate{N}) where N = print(io, "reflect($(summary(x.psi)))")
 print_block(io::IO, c::Concentrator) = print(io, "Concentrator: ", occupied_locs(c))
 print_block(io::IO, c::CachedBlock) = print_block(io, content(c))
-print_block(io::IO, c::Sum) = printstyled(io, "sum"; bold=true, color=color(ChainBlock))
+print_block(io::IO, c::Sum) = printstyled(io, "+"; bold=true, color=color(Sum))
```

Comment:
explain the change of title? I think we should follow the same convention that printed name is just the same with type name/constructor.

Reply:
`+` is more consistant with Julialang, `sum` takes vector `[]` as input.